tests/federation_redaction_test.go

```diff
@@ -0,0 +1,87 @@
+package tests
+
+import (
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/docker"
+	"github.com/matrix-org/complement/internal/federation"
+	"github.com/matrix-org/gomatrixserverlib"
+	"os"
+	"testing"
+	"time"
+)
+
+// test that a redaction is sent out over federation even if we don't have the original event
+func TestFederationRedactSendsWithoutEvent(t *testing.T) {
+	os.Setenv("COMPLEMENT_DEBUG", "1")
```

Comment:
I know this is a draft but I think you don't want this when we come to merge the test.

Reply:
D'oh! Thanks.